java/src/jmri/jmrix/loconet/LnOpsModeProgrammer.java

```diff
@@ -544,6 +558,18 @@ void initiializeBdOpsAccessTimer() {
         }
     }
 
+    void initiializeSV2AccessTimer() {
```

Comment:
**initiialize** spells **initialize** in English
I'm also a non native english developer so I struggle with this myself  :)